docs/js/snackbar.js

```diff
@@ -0,0 +1,60 @@
+/**Snackbar JS**/
+
+// Snackbar button selector id
+const leading = document.querySelector('#leading')
+const basline = document.querySelector('#basline')
+const stacked = document.querySelector('#stacked')
+
+// Snackbar div selector id
+const leadingClass = document.querySelector('#leadingClass')
+const baslineClass = document.querySelector('#baslineClass')
+const stackedClass = document.querySelector('#stackedClass')
+
+// Close button selector id
+const leadingClose = document.querySelector('#leadingClose')
+const baslineClose = document.querySelector('#baslineClose')
+const stackedClose = document.querySelector('#stackedClose')
+
+// Show snackbar EventListener
+leading.addEventListener('click', showLeading)
+basline.addEventListener('click', showBasline)
+stacked.addEventListener('click', showStacked)
+
+// Close snackbar EventListener
+leadingClose.addEventListener('click', closeLeading)
+baslineClose.addEventListener('click', closeBasline)
+stackedClose.addEventListener('click', closeStacked)
+
+// Show clicked snackbar
+function showLeading() {
+  leadingClass.classList.remove('hidden')
+  setTimeout(() => {

```

Comment:
you can write this code like this

 setTimeout(() => leadingClass.classList.add('hidden') , 3000)